.pylintrc

```diff
@@ -135,7 +135,7 @@ dummy-variables-rgx=^\*{0,2}(_$|unused_|dummy_)
 [FORMAT]
 
 # Maximum number of characters on a single line.
-# DEFAULT:  max-line-length=80
+DEFAULT:  max-line-length=80
```

Comment:
it should be
`max-line-length=80`
instead of
`DEFAULT:  max-line-length=80`
?